package.json

```
@@ -17,42 +12,15 @@
     "express": "^4.17.1",
     "express-openid-connect": "^2.5.1",
     "express-session": "^1.17.2",
-    "fs": "*",
     "passport": "^0.5.0",
     "passport-auth0": "^1.4.1",
     "pg": "^8.7.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-router-dom": "^6.2.1",
-    "react-scripts": "4.0.3",
-    "web-vitals": "^1.1.2"
+    "eslint": "^7.11.0",
+    "fs": "*"
   },
   "scripts": {
-    "serve": "node server.js",
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "dev": "concurrently \"npm run serve\" \"npm run start\"",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "proxy": "http://localhost:5000",
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
+    "serve": "node server/server.js",
```

now that we've separated the `client` directory, this can just be `start`:

```suggestion
    "start": "node server/server.js",
```